xml/depl_maintenance.xml

```diff
@@ -508,6 +508,258 @@
+      <para>
+       The <filename>/etc/pg2mysql/databases.yaml</filename> file contains a
+       list of databases along with their source and target connection strings:
+      </para>
+<screen>keystone:
+  source: postgresql://keystone:vZn3nfxXzv97@192.168.243.87/keystone
+  target: mysql+pymysql://keystone:vZn3nfxXzv97@192.168.243.88/keystone?charset=utf8
+glance:
+  source: postgresql://glance:cOau7NhaA54N@192.168.243.87/glance
+  target: mysql+pymysql://glance:cOau7NhaA54N@192.168.243.88/glance?charset=utf8
+cinder:
+  source: postgresql://cinder:idRll2gJPodv@192.168.243.87/cinder
+  target: mysql+pymysql://cinder:idRll2gJPodv@192.168.243.88/cinder?charset=utf8</screen>
+     </step>
+     <step>
+      <para>
+       Install the <package>python-psql2mysql</package> package on the
+       &admnode; or directly on the &dbnode;.
+      </para>
+      <para>
+       Running the tool directly on &dbnode; speeds up the migration operation,
+       as this does not require moving database data over the network. You can
+       also use <filename>/etc/pg2mysql/databases.yaml</filename> as the input
+       file for the <systemitem>psql2mysql</systemitem> tool. This is the
+       recommended way of running the tool.
+      </para>
+      <para>
+       Running the tool on the &admnode; is advisable if the input batch file
+       is already located there, or if you prefer not to install additional
+       software on cluster nodes.
+      </para>
+     </step>
+     <step>
+      <para>
+       To determine whether the &postgres; databases contain data that cannot
+       be migrated to &mariadb;, run <command>psql2mysql</command> with the
+      <literal>precheck</literal> option:</para>
+<screen>psql2mysql \
+--source postgresql://keystone:vZn3nfxXzv97@192.168.243.87/keystone \
+--target mysql+pymysql://keystone:vZn3nfxXzv97@192.168.243.88/keystone?charset=utf8 \
+precheck</screen>
+      <para>
+       To run precheck operation on all databases in a single operation, use the
+       <literal>--batch</literal> option and the
+       <filename>/etc/pg2mysql/databases.yaml</filename> file as follows:
+      </para>
+<screen>psql2mysql --batch /etc/pg2mysql/databases.yaml precheck</screen>
+      <para>
+       If the precheck indicates that there is data that cannot be imported
+       into &mariadb;, modify the offending data manually to fix
+       the problems.
+      </para>
+     </step>
+     <step>
+      <para>
+       Stop chef-client services on the nodes, to prevent regular runs of
+       chef-client from starting database-related &ostack; services again. Stop
+       all &ostack; services that make use of the database to prevent them from
+       writing new data during the migration.
+      </para>
+      <para>
+       On an HA setup, shut down all services that make use of the
+       database. To do this, use the <command>crm</command> command for
+       example:
+      </para>
+<screen>crm resource stop nova-api</screen>
```

Comment:
We might need to be more specific here and tell users exactly what to stop...

Reply:
`crm resource stop $resource` where `$resource` is

`apache2
keystone
cinder-api
glance-api
neutron-server
swift-proxy
nova-api`

To be continued...

Reply:
@rhafer do you know which ceilometer, aodh and barbican services to stop?

Reply:
Fixed.